tasks/tools.py

```diff
@@ -49,12 +53,113 @@ def remove_from_do(id):
 
     session.execute("COMMIT")
 
+
+def extract_nulls_from_log(infile, outfile, relevant_fields):
+    STR_NULL = 'contain only NULL values: '
+    STR_ERROR = '[ERROR]'
+    STR_INFO = '[INFO]'
+    STR_EXCEPTIONS = 'exceptions'
+
+    relevant_fields = relevant_fields.split(',')
+    cols = {}
+
+    try:
+        os.remove(outfile)
+    except OSError:
+        pass
+
+    with open(outfile, 'a') as fout:
```

Comment:
You can put multiple `with`s in a single line

```suggestion
    with open(outfile, 'a') as fout, open(infile) as fin:
```